client/dashboard/sites/site/index.tsx

```diff
@@ -28,7 +28,8 @@ import EnvironmentSwitcher from './environment-switcher';
 
 function Site() {
 	const isDesktop = useViewportMatch( 'medium' );
-	const sites = useQuery( sitesQuery() ).data;
+	const [ isSwitcherOpen, setIsSwitcherOpen ] = useState( false );
+	const sites = useQuery( { ...sitesQuery(), enabled: isSwitcherOpen } ).data;
```

Comment:
Let's format this one for the consistency!

```
const { data: sites } = ...
```

Reply:
I agree